Source/Fuse.Controls.Native/Android/SystemUiVisibility.uno

```diff
@@ -0,0 +1,92 @@
+using Uno;
+using Uno.Collections;
+using Uno.Compiler.ExportTargetInterop;
+
+namespace Fuse.Controls.Native
+{
+	[ForeignInclude(Language.Java,
+		"android.graphics.Color",
+		"android.os.Build",
+		"android.view.View",
+		"android.view.WindowManager",
+		"android.view.Window",
+		"android.view.WindowManager.LayoutParams")]
+	//Requires android SDK version 16 or newer(Jelly Bean)
```

Comment:
We [already require API level 16](https://www.fusetools.com/docs/basics/supported-platforms), so this comment isn't really necessary.